src/v2/Components/NavBar/Menus/UserMenu.tsx

```diff
@@ -22,10 +22,13 @@ export const UserMenu: React.FC = () => {
   const { trackEvent } = useTracking()
   const { mediator, user } = useContext(SystemContext)
 
-  const trackClick = event => {
-    const link = event.target
+  const trackClick = (event: React.MouseEvent<HTMLElement, MouseEvent>) => {
+    const link = event.currentTarget
+
+    if (!(link instanceof HTMLAnchorElement)) return
```

Comment:
This acts as a typeguard ensuring that the element is inferred as an anchor with an href below.